mk/config.mk

```diff
@@ -449,6 +449,7 @@ CFG_CORE_BGET_BESTFIT ?= $(call cfg-one-enabled, CFG_WITH_PAGER CFG_LOCKDEP)
 # Enable support for detected undefined behavior in C
 # Uses a lot of memory, can't be enabled by default
 CFG_CORE_SANITIZE_UNDEFINED ?= n
+CFG_TA_SANITIZE_UNDEFINED ?= n
```

Comment:
Please add a 
`_make PLATFORM=vexpress-qemu_armv8a CFG_CORE_SANITIZE_UNDEFINED=y CFG_TA_SANITIZE_UNDEFINED=y`
line in `.github/workflows/ci.yml`.

Reply:
Done. I added also CFG_TEE_RAM_VA_SIZE=0x00400000 , otherwise it doesn't compile.

Reply:
Yes, option 2 makes the most sense to me too.

Reply:
Done